versions.gradle

```diff
@@ -0,0 +1,175 @@
+]
+
+versions.lifecycle = "2.3.0-rc01"
+libs.AndroidX.Lifecycle = [
+    runtime: "androidx.lifecycle:lifecycle-runtime-ktx:$versions.lifecycle",
+    livedata: "androidx.lifecycle:lifecycle-livedata-ktx:$versions.lifecycle",
+    viewmodel: "androidx.lifecycle:lifecycle-viewmodel-ktx:$versions.lifecycle",
+]
+
+versions.room = "2.3.0-beta01"
+libs.AndroidX.Room = [
+    common: "androidx.room:room-common:$versions.room",
+    runtime: "androidx.room:room-runtime:$versions.room",
+    compiler: "androidx.room:room-compiler:$versions.room",
+    ktx: "androidx.room:room-ktx:$versions.room",
+    testing: "androidx.room:room-testing:$versions.room",
+]
+
+libs.AndroidX.Work = [
+    runtimeKtx: "androidx.work:work-runtime-ktx:2.4.0"
+]
+
+versions.compose = "1.0.0-alpha11"
+versions.composesnapshot = ""
+libs.AndroidX.Compose = [
+    compiler: "androidx.compose.compiler:compiler:$versions.compose",
+    runtime: "androidx.compose.runtime:runtime:$versions.compose",
+    livedata: "androidx.compose.runtime:runtime-livedata:$versions.compose",
+    foundation: "androidx.compose.foundation:foundation:$versions.compose",
+    layout: "androidx.compose.foundation:foundation-layout:$versions.compose",
+    ui: "androidx.compose.ui:ui:$versions.compose",
+    material: "androidx.compose.material:material:$versions.compose",
+    animation: "androidx.compose.animation:animation:$versions.compose",
+    tooling: "androidx.compose.ui:ui-tooling:$versions.compose",
+    test: "androidx.compose.ui:ui-test-junit4:$versions.compose",
+]
+
+versions.androidxhilt = "1.0.0-alpha03"
+libs.AndroidX.Hilt = [
+    work: "androidx.hilt:hilt-work:$versions.androidxhilt",
+    compiler: "androidx.hilt:hilt-compiler:$versions.androidxhilt",
+]
+
+versions.dagger = "2.31.2"
+libs.Dagger = [
+    dagger: "com.google.dagger:dagger:$versions.dagger",
+    compiler: "com.google.dagger:dagger-compiler:$versions.dagger",
+]
+
+versions.hilt = "2.31.2-alpha"
+libs.Hilt = [
+    library: "com.google.dagger:hilt-android:$versions.hilt",
+    compiler: "com.google.dagger:hilt-android-compiler:$versions.hilt",
+    testing: "com.google.dagger:hilt-android-testing:$versions.hilt",
+    gradlePlugin: "com.google.dagger:hilt-android-gradle-plugin:$versions.hilt",
+]
+
+versions.retrofit = "2.9.0"
+libs.Retrofit = [
+    retrofit: "com.squareup.retrofit2:retrofit:$versions.retrofit",
+    gsonConverter: "com.squareup.retrofit2:converter-gson:$versions.retrofit",
+]
+
+versions.okhttp = "4.9.0"
+libs.OkHttp = [
+    okhttp: "com.squareup.okhttp3:okhttp:$versions.okhttp",
+    loggingInterceptor: "com.squareup.okhttp3:logging-interceptor:$versions.okhttp",
+]
+
+versions.coil = "1.1.1"
+libs.Coil = [
+    coil: "io.coil-kt:coil:$versions.coil"
+]
+
+versions.roomigrant = "master-SNAPSHOT"
+libs.Roomigrant = [
+    library: "com.github.chrisbanes.Roomigrant:RoomigrantLib:$versions.roomigrant",
+    compiler: "com.github.chrisbanes.Roomigrant:RoomigrantCompiler:$versions.roomigrant",
```

Comment:
Where can I find this version of Roomigrant?

Reply:
https://github.com/chrisbanes/roomigrant

It's just ports it over to AndroidX